src/test/scala/uk/gov/hmrc/test/ui/pages/base/DriverHelper.scala

```diff
@@ -41,6 +41,10 @@ trait DriverHelper {
 
   def continue(): Unit = clickById("submit")
 
+  def saveAndReturnToSummary(): Unit = clickById("save_and_return_to_summary")
```

Comment:
Should this method be in driver helper also below one

Reply:
it is in the driver helper class

Reply:
Done